engine/classes/ElggAccessCollection.php

```diff
@@ -114,9 +114,9 @@ public function getDisplayName() {
 	 */
 	public function save() {
 		if ($this->id > 0) {
-			return _elgg_services()->accessCollections->rename($this->id, $this->name);
+			return (bool) _elgg_services()->accessCollections->rename($this->id, $this->name);
```

Comment:
this is not needed as you fixed that the function only returns `bool`

Reply:
if you're scared that the function may someday return something else add a unit test to make sure it doesn't